jekyll/_cci2/workflows.md

```diff
@@ -97,19 +98,19 @@ The following example shows a workflow with four sequential jobs. The jobs run a
 The following `config.yml` snippet is an example of a workflow configured for sequential job execution:
 
 ```yaml
-workflows:
-  version: 2
-  build-test-and-deploy:
-    jobs:
+version: 2.1
+workflows: 
+  build-test-and-deploy: 
+    jobs: 
       - build
-      - test1:
```

Comment:
The `-`s have been removed from the job calls within the workflow. They are required